docker/README.md

```diff
@@ -0,0 +1,10 @@
+## Testing mjolnir with docker
```

Comment:
Calling this directory `docker/` feels like it focuses on  the "how" rather than the "what". This is a testing infrastructure, right? Why not call it `testing/`?